src/modules/sync/lib/ipc-handlers.ts

```diff
@@ -158,16 +166,94 @@ export async function pushArchive(
 		throw new Error( 'No token found' );
 	}
 
+	let isUploadingPaused = false;
+	const file = fs.createReadStream( archivePath );
+	const fileSize = fs.statSync( archivePath ).size;
+	const filename = path.basename( archivePath );
+
+	const cleanup = () => {
+		file.destroy();
+		file.close();
+	};
+
+	const attachmentId = await new Promise< string >( ( resolve, reject ) => {
+		const upload = new Upload( file, {
+			endpoint: `${ STUDIO_FILE_UPLOADS_ENDPOINT_BASE }/${ remoteSiteId }`,
+			chunkSize: STUDIO_FILE_UPLOADS_CHUNK_SIZE,
+			retryDelays: STUDIO_FILE_UPLOADS_RETRY_DELAYS,
```

Comment:
```suggestion
			endpoint: `https://public-api.wordpress.com/rest/v1.1/studio-file-uploads/${ remoteSiteId }`,
			chunkSize: 500000,
			retryDelays: [ 0, 1000, 3000, 5000, 10000, 25000 ],
```
Since these constants aren't being reused, I would argue it simplifies the code to inline them.

Reply:
Makes sense to me. Updated in [9a3e9da](https://github.com/Automattic/studio/pull/2212/commits/9a3e9daf457c1a8e1a40236ce8a100a1e6eb5c7b). If needed in future, we can move them to constants.